src/components/Editor.js

```diff
@@ -75,6 +75,15 @@ class Editor extends Component<Props, State> {
         onLoad={this.onEditorLoad}
         {...aceStyleProps}
         markers={this.getMarkers()}
+        commands={[
+          {
+            name: 'run',
+            bindKey: { win: 'Ctrl-Enter', mac: 'Command-Enter' },
+            exec: () => {
```

Comment:
this can be reduced to `exec: this.handleClick,`

Reply:
Got this on the browser's console.
```
Warning: Cannot update during an existing state transition (such as within `render` or another component's constructor). Render methods should be a pure function of props and state; constructor side-effects are an anti-pattern, but can be moved to `componentWillMount`
```